modules/paapi.js

```diff
@@ -197,9 +315,30 @@ export function markForFledge(next, bidderRequests) {
     bidderRequests.forEach((bidderReq) => {
       config.runWithBidder(bidderReq.bidderCode, () => {
         const {enabled, ae} = getFledgeConfig();
-        Object.assign(bidderReq, {fledgeEnabled: enabled});
+        Object.assign(bidderReq, {
+          fledgeEnabled: enabled,
+          paapi: {
+            enabled,
+            componentSeller: !!moduleConfig.componentSeller?.auctionConfig
+          }
+        });
         bidderReq.bids.forEach(bidReq => {
-          deepSetValue(bidReq, 'ortb2Imp.ext.ae', bidReq.ortb2Imp?.ext?.ae ?? ae);
+          // https://github.com/InteractiveAdvertisingBureau/openrtb/blob/main/extensions/community_extensions/Protected%20Audience%20Support.md
+          const igsAe = bidReq.ortb2Imp?.ext?.igs != null
+            ? bidReq.ortb2Imp.ext.igs.ae || 1
+            : null
+          const extAe = bidReq.ortb2Imp?.ext?.ae;
+          if (igsAe !== extAe && igsAe != null && extAe != null) {
+            logWarn(MODULE, `Bid request defines conflicting ortb2Imp.ext.ae and ortb2Imp.ext.igs, using the latter`, bidReq);
+          }
+          const bidAe = igsAe ?? extAe ?? ae;
+          if (bidAe) {
+            deepSetValue(bidReq, 'ortb2Imp.ext.ae', bidAe);
+            bidReq.ortb2Imp.ext.igs = Object.assign({
+              ae: bidAe,
+              biddable: 1
```

Comment:
Noticed on https://github.com/InteractiveAdvertisingBureau/openrtb/blob/main/extensions/community_extensions/Protected%20Audience%20Support.md, it states that the biddable field is 0 by default (indicating a buyer is not allowed).  Also saw in the prebid "fledge" slack channel (about a month back) that the biddable field default may get reversed to 1 instead.  Is there an update on the latest around this?

Reply:
AFAIK there's no concrete plan for the spec to change.